Neos.ContentGraph.PostgreSQLAdapter/src/Domain/Projection/HierarchyHyperrelationRecord.php

```diff
@@ -108,6 +94,28 @@ public function addChildNodeAnchor(
         $this->updateChildNodeAnchors($childNodeAnchors, $databaseConnection);
     }
 
+    public function addChildNodeAnchorAfterFirstCandidate(
```

Comment:
what is a "candidate"? can we again add documentation what's going on here? :) and maybe a short diagram if needed